client/idrepo/enduser/src/main/java/org/apache/syncope/client/enduser/pages/SelfPasswordReset.java

```diff
@@ -91,10 +91,7 @@ public SelfPasswordReset(final PageParameters parameters) {
             @Override
             protected void onSubmit(final AjaxRequestTarget target) {
                 boolean checked = true;
```

Comment:
@andrea-patricelli it seems that `checked` can be safely removed

Reply:
You're right. Going to remove